src/pages/RoomMembersPage.tsx

```diff
@@ -208,85 +170,100 @@ function RoomMembersPage(props) {
 
             result.push({
                 keyForList: String(accountID),
-                accountID: Number(accountID),
-                isSelected: _.contains(selectedMembers, Number(accountID)),
-                isDisabled: accountID === props.session.accountID,
-                text: props.formatPhoneNumber(PersonalDetailsUtils.getDisplayNameOrDefault(details)),
-                alternateText: props.formatPhoneNumber(details.login),
-                icons: [
-                    {
-                        source: UserUtils.getAvatar(details.avatar, accountID),
-                        name: details.login,
-                        type: CONST.ICON_TYPE_AVATAR,
-                    },
-                ],
+                accountID,
+                isSelected: selectedMembers.includes(accountID),
+                isDisabled: accountID === session?.accountID,
+                text: formatPhoneNumber(PersonalDetailsUtils.getDisplayNameOrDefault(details)),
+                alternateText: details.login ? formatPhoneNumber(details.login) : '',
+                icons: details.login
```

Comment:
I am trying to make out if this has an odd impact because of this change? I can see earlier we would still have `source` and `type` if `details.login` not set.